src/steps/ms-defender/machine/converters.ts

```diff
@@ -24,6 +42,12 @@ export function createMachineEntity(data: Machine): Entity {
         name: data.computerDnsName,
         computerDnsName: data.computerDnsName,
         displayName: data.computerDnsName as string,
+        rbacGroupId: data.rbacGroupId, // INT-7983
```

Comment:
Is the Jira ticket comment the standard? This being an open source project, it seems a bit strange.